app/jobs/ingest_job.rb

```diff
@@ -10,12 +10,14 @@ class IngestJob < ApplicationJob
 
   # @param [Hash] model_params
   # @param [Array<String>] signed_ids for the blobs
+  # @param [BackgroundJobResult] background_job_result
   # @param [Boolean] start_workflow if true, start accessionWF
   # @param [Boolean] assign_doi if true, adds DOI to Cocina obj
-  # @param [BackgroundJobResult] background_job_result
+  # @param [String] priority ('default') determines the relative priority used for the workflow.
+  #                                      Value may be 'low' or 'default'
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Metrics/MethodLength
-  def perform(model_params:, signed_ids:, background_job_result:, start_workflow: true, assign_doi: false)
+  def perform(model_params:, signed_ids:, background_job_result:, start_workflow: true, assign_doi: false, priority: 'default')
```

Comment:
Would it be worthwhile to use constant values for these priorities, or to check that they are correct, to help ensure they are being used consistently?

Reply:
We already are doing that via the openapi.yml.  If you used something other than default or low it would be an invalid request.